app/views/shopping_list/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1>Shopping list</h1>
+
+<table class="food-table">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Price</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @unused_foods.each do |food| %>
+      <tr>
+        <td><%= link_to food.name, food_path(food), class: 'food-name' %></td>
+        <td>$<%= food.price * food.quantity %></td>
+      </tr>
+    <% end %>
+  </tbody>
+  <tfoot>
+    <tr>
+      <td><strong>Total</strong></td>
+      <td><strong>$<%= @unused_foods.sum { |food| food.price * food.quantity } %></strong></td>
+    </tr>
```

Comment:
- [ ] In the General shopping list you should display the food that is missing from all recipes. (Example: You have a food apple with 20 quantities and in a recipe, you added food ingredient apple in 30 quantity means 10 is missing in your food list so you should display the apple food with the price of 10 quantities in the general shopping list). Currently, You're showing all food in the general shopping list, and if any food is added to recipes you're removing it from the shopping list. Please make sure to fix this issue.